src/components/title/title.styles.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { CSSProperties } from 'react';
+import { css } from '@emotion/react';
+import { UseEuiTheme, useEuiTheme } from '../../services';
+import {
+  euiTextBreakWord,
+  euiFontSizeFromScale,
+  euiLineHeightFromBaseline,
+  _EuiThemeFontScale,
+  _EuiThemeFontSizeMeasurement,
+} from '../../global_styling';
+import { EuiTitleSize } from './title';
+
+/**
+ * Theme-specific scale settings
+ */
+type _EuiThemeTitle = {
+  fontSize: _EuiThemeFontScale;
+  lineHeight: _EuiThemeFontScale;
+  fontWeight?: CSSProperties['fontWeight']; // Defaults to bold
+  letterSpacing?: string; // Not currently used in Amsterdam, but may be used in future themes
+};
+
+const amsterdamThemeTitles: {
```

Comment:
There shouldn't be anything theme-named specific in the components folder

```suggestion
const titles: {
```

Reply:
👍 If we're moving away from having fontSize and lineHeight be separate from one other I shouldn't even need this blob.

Reply:
https://github.com/elastic/eui/pull/5842/commits/958497352b3bbd0897c8e9f3e50c0ea4982f0923